Source/WebCore/page/ChromeClient.h

```diff
@@ -563,6 +563,8 @@ class ChromeClient {
 #if ENABLE(ACCESSIBILITY_ANIMATION_CONTROL)
     virtual void isAnyAnimationAllowedToPlayDidChange(bool /* anyAnimationCanPlay */) { };
 #endif
+    virtual void resolveAccessibilityHitTestForTesting(WebCore::FrameIdentifier, const IntPoint&, CompletionHandler<void(String)>&&) { };
```

Comment:
Can we make this pure virtual or call the CompletionHandler with an empty string in the empty client case?

Reply:
Yes that's a good call